import { CustomError } from "./CustomError";

export class DeleteTaskError extends CustomError{
    constructor(message = "Sua tarefa n√£o pode ser deletada.") {
        super(message, 500);
    }
} 